src/main/java/io/jenkins/plugins/ml/IPythonBuilder.java

```diff
@@ -110,6 +117,9 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath ws, @Nonnull Launc
                                     }
                             }
                             break;
+                        case py:
+                            listener.getLogger().println(StringUtils.stripStart(interpreterManager.invokeInterpreter(tempFilePath.readToString()), "%text"));
```

Comment:
I **think** this doesn't work with agents, unless `invokeInterpreter` is doing using a `Callable`. But that's for a future issue I guess. Approving! :rocket:

Reply:
I ran with an agent connected in my local machine, it works fine if the file exists in the agent workspace. If it is in the master's workspace, it throws NoSuchFile Error. We can do the issue in our next release :heart: